_data/backup.yml

```
@@ -136,7 +136,14 @@ websites:
     twitter: multcloud
     facebook: multcloud
     img: multcloud.png
-
+    
+  - name: Nextcloud
+    url: https://www.nextcloud.com
+    img: nextcloud.png
+    tfa:
+      - totp
```

```suggestion
      - totp
      - u2f
```
According to the docs, ` u2f` keys are supported.